### ISSUE TITLE
Asteroid chunk recipe from Stopped at Nauvis

### ISSUE THREAD

**Issue by jxu** [factoriolab/factoriolab#1636]
I have just unlocked the space platform, so all the asteroid chunks come from the asteroid collector when the platform is stopped at Nauvis.
There's no recipe for this, only for space routes, solar system edge, and shattered planet?

https://factoriolab.github.io/spa/list?z=eJwNyMFKw0AUQNG.mcWFQtqCaRezaJJ586ZSUWwDuhEjBRudVgeU4OJ9u12czbn4TllUbvC8uHKcfPtIu-aa3RvdSGgIHUEs7JB75Bl5RzLybVKQH2RClsQNUYlb4oE4WvwgfhEntEMTeoceTHv0CT2ZjujZ9IIWtEIX6I1pTQqWhLQl9ZZeSZ-WMvXZ6onVidUf64FZodmMV0uapnc5F.9gtxZsb-3O5eHoK.fr5.N.p309.g__&v=11

The reason it's important to me is that oxide chunks seem to be very rare. 
https://www.reddit.com/r/factorio/comments/1gba0ox/oxide_asteroids_disproportionately_rare_or_just/ 
https://www.reddit.com/r/factorio/comments/1gt9puw/cant_seem_to_get_enough_oxide_asteroids/

**Comment by dcbroad3** [comment]
I don't think this is a bug, I'll switch it to a feature request, and I'll look into it when I can
